test/extensions/filters/http/ext_proc/filter_test.cc

```diff
@@ -2261,6 +2272,90 @@ TEST_F(HttpFilterTest, FailOnInvalidHeaderMutations) {
   filter_->onDestroy();
 }
 
+class HttpFilter2Test : public HttpFilterTest, public Http::HttpConnectionManagerImplMixin {};
+
+// Test proves that when onData(data, end_stream) is called before headers response is returned,
+// ext_proc filter will make the data buffered in ActiveStream buffer without triggering a buffer
+// over high watermark call, which ends in an 413 error return on request path, or a 500 error on
+// response path.
+TEST_F(HttpFilter2Test, LastOnDataCallExceedsStreamBufferLimitWouldJustRaiseHighWatermark2) {
```

Comment:
RaiseHighWatermark2? What does the "2" mean?

Reply:
ah, thanks for the catch. 
There used to be a 1.